content/news/2024-19-08-GalaxyInResearch/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Galaxy in Research: Advancing quantum materials research with Galaxy"
+tease: "At Oak Ridge National Laboratory, scientists have harnessed the power of Galaxy to bring together cutting-edge computing resources, paving the way for new discoveries in quantum materials through advanced neutron scattering data analysis."
+hide-tease: false
+authors: "Natalie Whitaker-Allen"
+date: "2024-08-19"
+tags: ["highlight"]
+subsites: [global,all]
+---
+
+# **Galaxy in Research: Advancing quantum materials research with Galaxy**
+
+*At Oak Ridge National Laboratory, scientists have harnessed the power of Galaxy to bring together cutting-edge computing resources, paving the way for new discoveries in quantum materials through advanced neutron scattering data analysis.*
+
+## **Research Overview**
+
+Understanding the behavior of correlated quantum materials involves intricate data analysis and sophisticated computational methods. Traditional approaches, such as density functional theory and linear spin-wave theory, often need to catch up in capturing the complexities of strong electron-electron interactions and localized electronic states. To address these challenges, researchers at Oak Ridge National Laboratory (ORNL) utilized advanced methods like dynamical mean-field theory (DMFT) and the dynamic cluster approximation (DCA) implemented in ORNL’s DCA++ code.
+
+The study focused on neutron scattering data analysis for quantum materials, which is crucial for revealing the magnetic excitations of these systems. By integrating Galaxy with high-performance computing (HPC) and cloud resources, the research team developed a fully automated end-to-end workflow to streamline this complex analysis.
+
+## **Key Findings**
+
+* **Advanced Computational Approach**: The team used DMFT and DCA methods to model correlated quantum materials, providing a detailed and accurate description of magnetic excitations.
+
+* **Workflow Automation:** The integration of Galaxy allowed for the automation of the DCA++ workflow, reducing the need for manual intervention and facilitating reproducibility.
+
+* **Resource Efficiency:** The workflow used HPC and cloud-based resources, demonstrating flexibility and scalability in handling large datasets and complex calculations.
+
+* **Scientific Impact:** The final results, including a comprehensive analysis of dynamic magnetic spin susceptibility, offer valuable insights into quantum materials, with potential implications for future neutron scattering experiments.
+
+![GalaxyInResearch_Watson](https://github.com/user-attachments/assets/ec1052af-2432-47f8-9c90-a18544f34064)
```

Comment:
@natalie-wa We probably want this to be a static upload, not a github attachment?

Reply:
@dannon Oh! Whoops - I was trying a different image method I had seen; I'm sorry about that! Didn't mean to cause an issue.